src/Microsoft.AspNet.Mvc.TagHelpers/Internal/JavaScriptStringEncoderExtensions.cs

```diff
@@ -0,0 +1,45 @@
+// Copyright (c) Microsoft Open Technologies, Inc. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System.Collections.Generic;
+using System.Text;
+using Microsoft.AspNet.WebUtilities.Encoders;
+
+namespace Microsoft.AspNet.Mvc.TagHelpers.Internal
+{
+    /// <summary>
+    /// Utility methods for dealing with JavaScript.
+    /// </summary>
+    public static class JavaScriptStringEncoderExtensions
+    {
+        /// <summary>
+        /// Encodes a .NET string array for safe use as a JavaScript array literal, including inline in an HTML file.
+        /// </summary>
+        public static string JavaScriptStringArrayEncode(
+            this IJavaScriptStringEncoder encoder,
+            IEnumerable<string> values)
```

Comment:
tend to avoid extension methods except in extreme situations e.g. when they are needed as part of a `public` API.  can we use a regular `static` method here?